src/Form/test-cases/usps_signup.html

```diff
@@ -301,7 +301,7 @@ <h3 class="subhead subhead-lg">Step 4: <strong>Enter your contact information</s
 				<div class="form-group" id="form-group-tmI">
 					<label id="for-tmI" for="tmI" class="control-label">M.I.</label>
 
-					<input type="text" name="tmI" size="1" maxlength="1" value="" tabindex="27" id="tmI" class="form-control" data-original-title="" title="" data-manual-scoring="middleName">
+					<input type="text" name="tmI" size="1" maxlength="1" value="" tabindex="27" id="tmI" class="form-control" data-original-title="" title="">
```

Comment:
This was marked as `middleName`, but in fact they wanted the initial only, which resulted in an invalid form when autofilled. With this update we're skipping 1-character input fields so this is now ignored, which I think it's correct.